app/javascript/components/enrollment/Enrollment.js

```diff
@@ -34,12 +34,6 @@ class Enrollment extends React.Component {
         blocked_sms: this.props.blocked_sms,
       },
     };
-    this.setEnrollmentState = debounce(this.setEnrollmentState.bind(this), 1000);
```

Comment:
I don't know that we wanted to remove the debouncing here.

Reply:
so I'm trying to figure out the syntax to do this bind in line to match the rest... I'm either getting an enrollmentState is not defined error with this when I try to push:

```  setEnrollmentState = () =>
    debounce(() => {
      let currentEnrollmentState = this.state.enrollmentState;
      this.setState({
        enrollmentState: {
          patient: { ...currentEnrollmentState.patient, ...enrollmentState.patient },
          propagatedFields: { ...currentEnrollmentState.propagatedFields, ...enrollmentState.propagatedFields },
          isolation: Object.prototype.hasOwnProperty.call(enrollmentState, 'isolation') ? !!enrollmentState.isolation : currentEnrollmentState.isolation,
          blocked_sms: enrollmentState.blocked_sms,
        },
      });
    }, 1000);```

Reply:
anyone have any idea how to do this?

Reply:
classic friday brain... accidentally removed the enrollmentState variable that gets passed in.  this should be good now